packages/core/src/input/KeyInputManager.ts

```diff
@@ -4,6 +4,11 @@ export class KeyInputManager {
   constructor() {
     document.addEventListener("keydown", this.onKeyDown.bind(this));
     document.addEventListener("keyup", this.onKeyUp.bind(this));
+    window.addEventListener("blur", this.handleUnfocus.bind(this));
+  }
+
+  private handleUnfocus(_event: FocusEvent): void {
+    this.keys = new Map<string, boolean>();
```

Comment:
I think using `this.keys.clear()` is safer here in case some code keeps a reference to the original map (even if it doesn't right now).

Reply:
done